training/src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { Error, Input } from './style';
+
+const TextField = (props) => {
```

Comment:
disabled prop is missing.
all the 3 text fields are of error variant on rendering.

Reply:
done